src/app/rmmapi/rmmauthguard.service.ts

```diff
@@ -100,10 +119,10 @@ export class RMMAuthGuardService implements CanActivate, CanActivateChild {
         this.router.navigate(['login']);
     }
 
-  redirectToSubscriptions() {
-      if (this.router.url !== 'account/subscriptions') {
-        console.log('Will navigate back to account/subscriptions');
-        this.router.navigate(['account/subscriptions']);
-      }
-    }
+    // redirectToSubscriptions() {
```

Comment:
Also, is this staying or should we remove it?

Reply:
thats rubbish, will remove